src/styles/blocks/main.scss

```diff
@@ -0,0 +1,32 @@
+.catalog {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 200px);
```

Comment:
You have hard-coded the width of the grid column (200px) which could potentially lead to issues if the content within the grid cells needs more space. Consider using minmax function to set a minimum and maximum width. This will make your grid more flexible. For example: `grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));`